### ISSUE TITLE
smokecurrent.sh does nothing but give a "Use of uninitialized value …" warning

### ISSUE THREAD

**Issue by pjacklam** [abeltje/Test-Smoke#77]
I have run
```
$ perl /usr/local/bin/configsmoke.pl
```
and regardless of whether I explictly specify a hostname or not (see issue https://github.com/abeltje/Test-Smoke/issues/76) and specify a timeout or not, the script `smokecurrent.sh` doesn't work. The script creates a lockfile, but immediately removes the lockfile and exits. The log shows that something isn't right:
```
$ ./smokecurrent.sh
$ cat smokecurrent.log
Use of uninitialized value in sprintf at /usr/local/lib/perl5/Test/Smoke/App/Base.pm line 393.
Invalid value '' for option 'ua_timeout'
```

**Comment by abeltje** [comment]
So the shell-script is a wrapper around `tssmokeperl.pl` which is a wrapper around:
* `tssynctree.pl`
* `tsrunsmoke.pl`
* `tsreport.pl`
* `tssendrpt.pl`
* `tsarchive.pl`

with no actual log-data in the logfile, one could try to run these scripts in order (don't forget `-c smokecurrent` as extra switch on the command line).

Please let me know how this works out,

thanks!

**Comment by abeltje** [comment]
This should be resolved in version 1.82
